worker/Dockerfile.template

```diff
@@ -47,4 +47,4 @@ COPY --from=node-build /tmp/node/package.json .
 COPY --from=node-build /tmp/node/node_modules node_modules
 COPY --from=node-build /tmp/node/build build
 
-CMD [ "./entry.sh" ]
+CMD ./entry.sh
```

Comment:
y tho? the former is the preferred syntax

Reply:
@klutchell This is to execute the command with a shell, which results in our messages in the entry script being printed to stdout. With the JSON array format, the script is executed without a shell, and the messages aren't printed.